components/error_page/workspace_user_limit_reached.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import computerWithAlert from 'images/cloud/computer-with-alert.svg';
+import mattermostCloud from 'images/cloud/mattermost-cloud.svg';
+import wavesBackground from 'images/cloud/waves.svg';
+import blueDotes from 'images/cloud/blue.svg';
```

Comment:
Should probably be blueDots not blueDotes.

Reply:
yes, thanks for finding this typo.